workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.ts

```diff
@@ -271,6 +273,24 @@ export default class DynamoDBService {
     return query;
   }
 
+  /**
+   * Places a DDB Update call and formats the response.
+   *
+   * @param updateParams - {@link UpdateParams} object to pass to the update request
+   * @returns a {@link UpdateUnmarshalledOutput} item where Attributes is unmarshalled
+   */
+  public async updateExecuteAndFormat(updateParams: UpdateParams): Promise<UpdateUnmarshalledOutput> {
+    const result = await this.update(updateParams).execute();
+
+    const unmarshalledResult = {
+      Attributes: result.Attributes ? unmarshall(result.Attributes) : undefined,
+      ConsumedCapacity: result.ConsumedCapacity || undefined,
```

Comment:
I would consider replacing `||` with `??`. Using the `OR` operator can lead to bugs where if `ConsumedCapacity` is a number and equals 0, it will set the value to `undefined`. The [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) only uses the right hand side if the left hand side is `undefined` or `null`